src/bundle/Resources/public/css/button.css

```diff
@@ -0,0 +1,83 @@
+.ez-button {
+    display: inline-block;
+    line-height: normal;
+    white-space: nowrap;
+    vertical-align: middle;
+    text-align: center;
+    cursor: pointer;
+    -webkit-user-drag: none;
+    user-select: none;
+    box-sizing: border-box;
+
+    text-decoration: none;
+    padding: 0.5em 1em;
```

Comment:
In what part of the application are you implementing this? Do you have a screenshot?
`ez-button` is the default value, so there are specific classes, maybe we can specify this.